clio-client/src/main/scala/org/broadinstitute/clio/client/metadata/CramMover.scala

```diff
@@ -0,0 +1,288 @@
+          destination,
+          newBasename.map(_ + CramExtensions.InsertSizeHistogramPdfExtension)
+        )
+      ),
+      insertSizeMetricsPath = src.insertSizeMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.InsertSizeMetricsExtension)
+        )
+      ),
+      qualityDistributionPdfPath = src.qualityDistributionPdfPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.QualityDistributionPdfExtension)
+        )
+      ),
+      qualityDistributionMetricsPath = src.qualityDistributionMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.QualityDistributionMetricsExtension)
+        )
+      ),
+      rawWgsMetricsPath = src.rawWgsMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.RawWgsMetricsExtension)
+        )
+      ),
+      readgroupAlignmentSummaryMetricsPath = src.readgroupAlignmentSummaryMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.ReadGroupAlignmentMetricsExtension)
+        )
+      ),
+      readgroupGcBiasPdfPath = src.readgroupGcBiasPdfPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.ReadGroupGcBiasPdfExtension)
+        )
+      ),
+      readgroupGcBiasDetailMetricsPath = src.readgroupGcBiasDetailMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.ReadGroupGcBiasDetailMetricsExtension)
+        )
+      ),
+      readgroupGcBiasSummaryMetricsPath = src.readgroupGcBiasSummaryMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.ReadGroupGcBiasSummaryMetricsExtension)
+        )
+      ),
+      recalDataPath = src.recalDataPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.RecalDataExtension)
+        )
+      ),
+      baitBiasDetailMetricsPath = src.baitBiasDetailMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.BaitBiasDetailMetricsExension)
+        )
+      ),
+      baitBiasSummaryMetricsPath = src.baitBiasSummaryMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.BaitBiasDetailMetricsExension)
+        )
+      ),
+      wgsMetricsPath = src.wgsMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.WgsMetricsExension)
+        )
+      ),
+      readgroupLevelMetricsFiles =
+        src.readgroupLevelMetricsFiles.map(_.map(buildFilePath(_, destination)))
```

Comment:
no renaming required here?

Reply:
Nope. If we did rename we'd get naming conflicts all over the place (unless we did something like `newBase + full old name`)